demo/components/demo/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,30 @@
+<span class="slds-dropdown-trigger slds-dropdown-trigger--click" nglDropdown [(open)]="open">
+  <button class="slds-button slds-button--neutral" nglDropdownTrigger>
+    <span class="slds-truncate">Select an item</span>
+    <svg aria-hidden="true" class="slds-button__icon slds-button__icon--hint">
```

Comment:
Can you use `ngl-icon`? This way you will not need to take extra care to work on different environments, locally, gh-pages, plunker etc